app/src/main/java/dev/aldi/sayuti/editor/view/item/ItemBottomNavigationView.java

```diff
@@ -0,0 +1,79 @@
+package dev.aldi.sayuti.editor.view.item;
+
+import a.a.a.sy;
+import a.a.a.wB;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import com.besome.sketch.beans.ViewBean;
+
+public class ItemBottomNavigationView extends LinearLayout implements sy {
+
+    /* renamed from: a  reason: collision with root package name */
+    public ViewBean f12a;
```

Comment:
You used JaDX to decompile the files. And it automatically renamed a field. This will not work, unless you edit all references to that field.